packages/orchestration/src/exos/chain-hub.js

```diff
@@ -223,38 +293,48 @@ export const makeChainHub = (agoricNames, vowTools) => {
      * @param {IBCConnectionInfo} connectionInfo
      */
     registerConnection(chainId1, chainId2, connectionInfo) {
-      const key = connectionKey(chainId1, chainId2);
-      connectionInfos.init(key, connectionInfo);
+      const [key, normalized] = normalizeConnectionInfo(
+        chainId1,
+        chainId2,
+        connectionInfo,
+      );
+      connectionInfos.init(key, normalized);
     },
 
     /**
-     * @param {string | { chainId: string }} chain1
-     * @param {string | { chainId: string }} chain2
+     * @param {string | { chainId: string }} primary
+     * @param {string | { chainId: string }} counter
```

Comment:
```suggestion
     * @param {string | { chainId: string }} primary the primary chain
     * @param {string | { chainId: string }} counter the counterparty chain
```

counter == counterparty may not be immediately obvious